Assets/Scripts/UI/Interaction Wheel/InteractableWheel.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InteractableWheel : MonoBehaviour {
```

Comment:
Better class name might be InteractionWheel, since the folder in which it resides is Interaction Wheel